Makefile

```diff
@@ -139,8 +138,12 @@ doctest: venv
 	@echo "Testing of doctests in the sources finished, look at the " \
 	      "results in $(BUILDDIR)/doctest/output.txt."
 
+htmlview: html
+	$(PYTHON) -c "import os, webbrowser; webbrowser.open('file://' + os.path.realpath('_build/html/index.html'))"
+
 check:
 	$(PYTHON) tools/rstlint.py -i tools -i venv
 
-serve: html
-	tools/serve.py _build/html
+serve:
+	@echo "The serve target was removed, use htmlview instead" \
```

Comment:
Does this need an extra space between `instead` and `"`?
`linkcheck` and `doctest` have it, but `latex`, `qthelp`, and `htmlhelp` don't.

Maybe add quotes around `'serve'` and  `'htmlview'`.

Reply:
Quotes added.

It doesn't need an extra space.

Without:

```
The 'serve' target was removed, use 'htmlview' instead (see https://github.com/python/cpython/issues/80510)
```

With:

```
The 'serve' target was removed, use 'htmlview' instead  (see https://github.com/python/cpython/issues/80510)
```